<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LayoutEditorWidget</class>
 <widget class="QWidget" name="LayoutEditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_7">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="LevelViewWidget" name="levelViewWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_6" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QFrame" name="frame_5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="controlNodeLabel">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Control node</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="AddRemoveListWidget" name="controlNodeList" native="true"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QWidget" name="widget_7" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QPushButton" name="nodeUpButton">
                <property name="text">
                 <string>^</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="nodeDownButton">
                <property name="text">
                 <string>v</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="NodeEditorStackedWidget" name="nodeEditorStackedWidget"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="AddRemoveListWidget" name="spawnList" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="PatternListWidget" name="patternListWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>224</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QWidget" name="widget_4" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QToolButton" name="modifyToolButton">
              <property name="text">
               <string>M</string>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/lonely/icons/icon_pencil.png</normaloff>:/lonely/icons/icon_pencil.png</iconset>
              </property>
              <property name="shortcut">
               <string>Z</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="copyToolButton">
              <property name="text">
               <string>C</string>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/lonely/icons/icon_clone.png</normaloff>:/lonely/icons/icon_clone.png</iconset>
              </property>
              <property name="shortcut">
               <string>X</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="moveToolButton">
              <property name="text">
               <string>D</string>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/lonely/icons/icon_move.png</normaloff>:/lonely/icons/icon_move.png</iconset>
              </property>
              <property name="shortcut">
               <string>C</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QWidget" name="insertOverwriteWidget" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <widget class="QRadioButton" name="overwriteButton">
                <property name="text">
                 <string>Overwrite</string>
                </property>
                <property name="shortcut">
                 <string>A</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="insertButton">
                <property name="text">
                 <string>Insert</string>
                </property>
                <property name="shortcut">
                 <string>S</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_8" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QCheckBox" name="copyNodesButton">
                <property name="text">
                 <string>Copy nodes</string>
                </property>
                <property name="shortcut">
                 <string>D</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Zoom</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="StandardZoomComboBox" name="zoomBox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QWidget" name="widget_5" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QCheckBox" name="gridBox">
                <property name="text">
                 <string>Grid</string>
                </property>
                <property name="shortcut">
                 <string>V</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="patternMarkersBox">
                <property name="text">
                 <string>Pattern markers</string>
                </property>
                <property name="shortcut">
                 <string>B</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="spawnsBox">
                <property name="text">
                 <string>Spawns</string>
                </property>
                <property name="shortcut">
                 <string>N</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="loopsBox">
                <property name="text">
                 <string>Jumps</string>
                </property>
                <property name="shortcut">
                 <string>M</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget" native="true">
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QRadioButton" name="visualButton">
                <property name="text">
                 <string>Visual</string>
                </property>
                <property name="shortcut">
                 <string>Q</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="behaviorButton">
                <property name="text">
                 <string>Behavior</string>
                </property>
                <property name="shortcut">
                 <string>W</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="solidityButton">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>Solidity</string>
                </property>
                <property name="shortcut">
                 <string>E</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="modifyToolAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>M</string>
   </property>
   <property name="toolTip">
    <string>Modify tool</string>
   </property>
  </action>
  <action name="copyToolAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>C</string>
   </property>
   <property name="toolTip">
    <string>Copy tool</string>
   </property>
  </action>
  <action name="moveToolAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>D</string>
   </property>
   <property name="toolTip">
    <string>Move/delete tool</string>
   </property>
  </action>
  <zorder></zorder>
  <zorder>widget_3</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PatternListWidget</class>
   <extends>QListWidget</extends>
   <header>patternlistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>StandardZoomComboBox</class>
   <extends>QComboBox</extends>
   <header>standardzoomcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>AddRemoveListWidget</class>
   <extends>QWidget</extends>
   <header>addremovelistwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LevelViewWidget</class>
   <extends>QWidget</extends>
   <header>levelviewwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NodeEditorStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header>nodeeditorstackedwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
